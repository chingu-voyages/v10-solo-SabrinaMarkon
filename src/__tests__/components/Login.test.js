import React from 'react';
import { shallow } from 'enzyme';

// components
import Login from '../../components/Login';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Login />);
});

describe('Testing Login component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('Login'));
  });
  it('it should match the Login snapshot', () => {
    // If snapshot exists, compare. If not, create the snapshot.
    expect(wrapper).toMatchSnapshot();
  });
});