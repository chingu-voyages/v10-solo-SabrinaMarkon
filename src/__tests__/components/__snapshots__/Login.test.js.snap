// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Login component it should match the Login snapshot 1`] = `undefined`;
